tools/travis/build-helm.sh

```diff
@@ -148,19 +151,20 @@ OW_CONTAINER_FACTORY=${OW_CONTAINER_FACTORY:="docker"}
 TIMEOUT_STEP_LIMIT=${TIMEOUT_STEP_LIMIT:=60}
 
 # Label invoker nodes (needed for DockerContainerFactory-based invoker deployment)
-echo "Labeling invoker node"
-kubectl label nodes --all openwhisk-role=invoker
+echo "Labeling kube-node-2 as an invoker node"
+kubectl label nodes kube-node-2 openwhisk-role=invoker
 kubectl describe nodes
 
 # Create namespace
 echo "Create openwhisk namespace"
 kubectl create namespace openwhisk
 
-# configure Ingress
+# configure a NodePort Ingress assuming kubeadm-dind-cluster conventions
 WSK_PORT=31001
-WSK_HOST=$(kubectl describe nodes | grep Hostname: | awk '{print $2}')
-if [ "$WSK_HOST" = "minikube" ]; then
-    WSK_HOST=$(minikube ip)
+WSK_HOST=$(kubectl describe node kube-node-1 | grep InternalIP: | awk '{print $2}')
```

Comment:
Can you explain why use `kube-node-1` here?

Reply:
added comment